api/desecapi/models.py

```diff
@@ -981,6 +982,21 @@ def _state_fields(self):
         return super()._state_fields + [self.user.email]
 
 
+class AuthenticatedChangeOutreachPreferenceUserAction(AuthenticatedEmailUserAction):
+    outreach_preference = models.BooleanField()
+
+    class Meta:
+        managed = False
+
+    @property
+    def _state_fields(self):
+        return super()._state_fields + [self.user.outreach_preference] + [self.outreach_preference]  # both needed?
```

Comment:
code should not get invalidated just because it has already been applied (idempotent)

Reply:
done (removed property)